aiida/manage/external/postgres.py

```diff
@@ -69,13 +69,15 @@ def from_profile(cls, profile, **kwargs):
         :returns: Postgres instance pre-populated with data from AiiDA profile
         """
         dbinfo = DEFAULT_DBINFO.copy()
-
-        get = profile.dictionary.get
         dbinfo.update(
             dict(
-                host=get('AIIDADB_HOST', DEFAULT_DBINFO['host']),
```

Comment:
actually, the docstring of the function explains why it is using only host and port

Reply:
I think the solution to the problem is to switch the way `verdi status` determines whether postgres is working.
The `Postgres` class is meant to connect to a postgres cluster as the superuser.
`verdi status` could simply go via `psycopg2` like `load_dbenv` does

Reply:
Aha ok, gotchya